test/Autocomplete.js

```diff
@@ -132,6 +132,47 @@ describe('Autocomplete', function() {
 		});
 	});
 
+	it('should select an option by pressing enter key', function(done) {
+		component = new Autocomplete({
+			data: filterData,
+			inputElement: input
+		});
+
+		input.value = 'Al';
+		simulateFocus(input);
+
+		async.nextTick(function() {
```

Comment:
I know that this test file is always using `async.nextTick` (that's because it's an old component), but that makes the tests harder to understand, since it's not clear what they're waiting for. I prefer using the component events (like `rendered` or `stateSynced`) instead for this, so that it's clear that we're waiting first for the `Autocomplete` render to finish, and then waiting for the `List` render. In fact I think that now we don't need to wait for `List` explicitly anymore, since it's rendered inside `Autocomplete`'s soy (it used to be rendered separately in the past, which I think is why we used to call `async.nextTick` twice instead of only once). Can you try replacing these with calls waiting for `Autocomplete`?